### PR TITLE
Change json URL according to Universal-Team

### PR DESCRIPTION
**Pull request by lifehackerhansol**
https://github.com/Universal-Team/db#using-the-data

### CODE REVIEW

**Comment by LightSage** [comment]
Most of this was written a while ago, which I just adapted it enough to work nicely for public usage.

File: main.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def get_random_app(self) -> dict:
 
 async def udb_cache_loop():
     while True:
-        url = "https://raw.githubusercontent.com/Universal-Team/db/master/docs/data/full.json"
+        url = "https://db.universal-team.net/data/full.json"
         resp = await app.state.session.get(url)
         r = json.loads(await resp.text())
```
Comment (LightSage) [inline main.py:76]: This should use ClientResponse.json() instead. i.e. `r = await resp.json()`

**Review by LightSage — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (lifehackerhansol) [reply main.py]: Amended.

**Review by LightSage — APPROVED** [review]

**Comment by lifehackerhansol** [comment]
Doesn't seem necessary to use json import after that, it isn't used anywhere.

**Pull request merged**